c2corg_api/scripts/loadtests/create_test_users.py

```diff
@@ -38,29 +35,27 @@ def main(argv=sys.argv):
     # create a fresh schema on the target database
     Session = sessionmaker(extension=ZopeTransactionExtension())  # noqa
     session = Session(bind=engine)
-    Base.metadata.drop_all(engine, checkfirst=True)
     setup_db(engine, session)
```

Comment:
`setup_db` tries to re-create the database schema. I guess you don't want to do that?